letsencrypt/client/client.py

```diff
@@ -43,7 +44,6 @@ class Client(object):
     """
     zope.interface.implements(interfaces.IAuthenticator)
 
-    Key = collections.namedtuple("Key", "file pem")
```

Comment:
Apparently @jdkasten has a separate fix for master that will make the CSR change too (and I think the documentation changes). So we will get that when that's merged.

The reason for moving it out of client.py had to do with client's import of the Apache configurator (making it hard to test other, non-Apache configurators in isolation) and the conceptual sense that these data type are more generally-applicable things that aren't "part of the client".